dpdata/system.py

```diff
@@ -1460,18 +1460,17 @@ def check_atom_names(self, system):
         if len(new_in_system):
             # A new atom_name appear, add to self.atom_names
             self.atom_names.extend(new_in_system)
-            self.atom_names.sort()
```

Comment:
I am wondering whether we need to sort atom names when self.atom_names does not have the enough element?